prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -768,6 +768,8 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             // connection are closed
             // after about 10 minutes
             config.maxLifetime = 180000
+            // Release connections after 60 seconds of idle
+            config.idleTimeout = 60000
```

Comment:
The docs have a very confusing line:

> Idle connections will not be retired once the pool reaches `minimumIdle` connections.

Does this mean that once we've got 2 idle connections we'll just keep accumulating connections??